### PR TITLE
FeatureEntryPoints

### PR DESCRIPTION
**Pull request by Autowinto**

### CODE REVIEW

File: jhotdraw-app/src/main/java/org/jhotdraw/app/action/file/LoadFileAction.java [hunk 1]
```diff
@@ -75,6 +77,8 @@ public class LoadFileAction extends AbstractSaveUnsavedChangesAction {
     /**
      * Creates a new instance.
      */
+
+    @FeatureEntryPoint("Loading File Formats - LoadFileAction - Constructor")
```
Comment (jfriisj) [inline jhotdraw-app/src/main/java/org/jhotdraw/app/action/file/LoadFileAction.java]: is this correct ?

**Comment by Busch31** [comment]
Several unused imports making maven build fail. Please fix this

**Review by jfriisj — APPROVED** [review]

**Review by mgertz — APPROVED** [review]
LGTM

**Review by Busch31 — APPROVED** [review]
looks good to me

**Review by KimjeLassen — APPROVED** [review]
LGTM

**Pull request merged**